sources/packages/backend/apps/api/src/route-controllers/education-program-offering/models/education-program-offering.dto.ts

```diff
@@ -95,16 +71,27 @@ export class EducationProgramOfferingAPIInDTO {
   offeringStatus: OfferingStatus;
   @Allow()
   offeringType: OfferingTypes;
-  @IsOptional()
+  @Allow()
```

Comment:
why allow is used instead of optional? Same doubt in other places as well

Reply:
Added also a comment to mention that the validation is handled by the `OfferingValidationModel`.